### DISCUSSION TITLE
Apparently Automatic1111 VAE Decode is still faster than Forge.

### DISCUSSION THREAD

**Discussion by AndreyRGW** [lllyasviel/stable-diffusion-webui-forge#526] [General]
Apparently A1111 VAE Decode is still faster than Forge (at least on sdxl models). On the same settings, on the same model, A1111 generates 2-3 seconds faster than Forge. The only difference is that I have FP8 cache enabled in A1111.

**Comment by Gourieff** [comment]
What GPU do you use? 
With Forge we have a huge speed boost up to 2-3 times faster than in A1111
In my case it takes ~2 sec to generate 768x960 with LCM SD15 models (~7 it/s, RTX4060Ti 16Gb)

**Reply by AndreyRGW** [reply]
> What GPU do you use? With Forge we have a huge speed boost up to 2-3 times faster than in A1111 In my case it takes ~2 sec to generate 768x960 with LCM SD15 models (~7 it/s, RTX4060Ti 16Gb)

I'm using RTX 2060 12GB.

In my case Forge generates images with SDXL models 2-3 seconds longer, e.g. where A1111 generates in 10.3 seconds, Forge generates in 12.6. 
I decided to try SD15 LCM model and the situation was the same: I used 768x960 resolution like you and A1111 handled it in 3.3 seconds and Forge in 5.9 seconds.

**Reply by Kenjiv3** [reply]
> > What GPU do you use? With Forge we have a huge speed boost up to 2-3 times faster than in A1111 In my case it takes ~2 sec to generate 768x960 with LCM SD15 models (~7 it/s, RTX4060Ti 16Gb)
> 
> I'm using RTX 2060 12GB.
> 
> In my case Forge generates images with SDXL models 2-3 seconds longer, e.g. where A1111 generates in 10.3 seconds, Forge generates in 12.6. I decided to try SD15 LCM model and the situation was the same: I used 768x960 resolution like you and A1111 handled it in 3.3 seconds and Forge in 5.9 seconds.

Are you counting from the moment you press "Generate" or from the moment you start generating the "0% appears and increases" image?
I think you're counting from the moment you press Generate until it's done.
With Forge, when you switch checkpoints, they won't be loaded into DRAM right away, but when you press Generate, the process of loading into DRAM starts.
